Reachability.m

```diff
@@ -98,8 +98,9 @@ +(instancetype)reachabilityWithHostname:(NSString*)hostname
     SCNetworkReachabilityRef ref = SCNetworkReachabilityCreateWithName(NULL, [hostname UTF8String]);
     if (ref) 
     {
-        id reachability = [[self alloc] initWithReachabilityRef:ref];
-
+        id reachability = [[self alloc] initWithReachabilityRef:CFBridgingRetain((__bridge id)ref)];
+	CFRelease(ref);
```

Comment:
As much as the analyzer doesn't report possible problems, I don't really care what solution is bes among those two. They both look fine to me.
However don't blame others if you don't read code carefully... ;-)